src/en/proposals/pjb-multi-character-slot.md

```diff
@@ -0,0 +1,60 @@
+# Multiple Character Slots
+
+| Designers | Implemented | GitHub Links |
+|---|---|---|
+| PJB3005 | :x: No | TBD |
+
+## Background
+
+Currently, a player can have one character slot selected. This controls their character's appearance, name, job preferences, and antag preferences. This is relatively restrictive, as people may want to have different settings per job. For example:
+
+* Some people prefer having different characters per job or department, for roleplay reasons.
+* The work on [loadouts](https://github.com/space-wizards/space-station-14/pull/25715) makes it so that loadouts are assigned per-job, which is kind of clunky.
+* Some "jobs" like clown or borg shouldn't use your normal character name. In SS14 this is currently suffering, in SS13 there's either a popup box when you spawn to pick your name OR multiple extra fields in the character info for "character name as clown" and such. Both suck.
+
+## Proposal
+
+It should be possible to have multiple character slots active at once. When selecting jobs, a job is picked for all your characters at once, after which the actual character slot is picked.
```

Comment:
I think this is a really clunky solution that isn't very clear to the player how it works.